pkg/agent/qrm-plugins/memory/dynamicpolicy/policy_test.go

```diff
@@ -3418,3 +3427,132 @@ func TestDynamicPolicy_adjustAllocationEntries(t *testing.T) {
 		})
 	}
 }
+
+func BenchmarkGetTopologyHints(b *testing.B) {
+	klog.SetOutput(ioutil.Discard)
+	klog.V(0)
+	klog.LogToStderr(false)
+	topoCases := []topoTestCase{
+		//{
```

Comment:
maybe it's better not to keep those commented codes

Reply:
use b.ResetTimer() before real benchmark code